docs/filter_kgraph_std_dev.md

```diff
@@ -29,40 +29,43 @@ properties:
     description: The name of the edge attribute to filter on.
     example: normalized_google_distance
     type: string
-  edge_keys:
-    default: null
+  keep_top_or_bottom:
+    default: top
+    description: Indicate whether or not the threshold should be placed in the top
+      or bottom of the values. E.g. top set as True will set the cutoff for filtering
+      as the mean + threshold * std_dev while setting top to False will set the cutoff
```

Comment:
This seems a little odd that the `keep_top_or_bottom` isn’t controlling whether the top or bottom is kept, but rather if it’s + or - the std dev.

Reply:
Perhaps pick plus_or_minus_std_dev for this name instead